test/testlib/OpenSSL/Test.pm

```diff
@@ -460,6 +460,7 @@ sub run {
     # to make it easier to compare with a manual run of the command.
     if ($opts{capture} || defined($opts{prefix})) {
 	my $pipe;
+	my $_;
```

Comment:
Shouldn't that be `local` rather than `my`? I'm not sure perl associates the "magic" $_ with the locally lexical one...